config/environments/development.rb

```diff
@@ -6,10 +6,10 @@
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
+  config.cache_classes = true
```

Comment:
I prefer setting this to false most of the time.  It slows down development to have to reset the server on each code change.  We can always set it to `true` for rails upgrades and such.  
```suggestion
  config.cache_classes = false
```